test/test_torch.py

```diff
@@ -16706,6 +16417,216 @@ def caller(cls,
     for test in tensor_op_tests:
         caller(cls, *test)
 
+def _generic_input_fn(dtype, device):
+    input = []
+    input.append(list(range(-5, 5)))
+    input.append([0 for x in range(-5, 5)])
+    input.append([x + 1e-6 for x in range(-5, 5)])
```

Comment:
No symmetric -1e-6 like 1e10 below?

Reply:
Symmetric 1e10 gives large positive and large negative values, here -1e-6 would give values in the same range, so probably does not make sense. Copy pase from original